frontend/public/components/utils/kebab.tsx

```diff
@@ -207,9 +207,9 @@ export type KebabOption = {
   href?: string, callback?: () => any;
   accessReview?: AccessReviewResourceAttributes;
 };
-export type KebabAction = (kind: K8sKind, obj: K8sResourceKind, resources?: any) => KebabOption;
+export type KebabAction = (kind: K8sKind, obj: K8sResourceKind, resources?: any, customData?: any) => KebabOption;
 
-export type KebabOptionsCreator = (kindObj: K8sKind, obj: K8sResourceKind, resources?: any) => KebabOption[];
+export type KebabOptionsCreator = (kindObj: K8sKind, obj: K8sResourceKind, resources?: any, customData?: any) => KebabOption[];
```

Comment:
This type duality isn't necessary IMO, please see https://github.com/openshift/console/pull/1907#issuecomment-510536568.

Reply:
it might be necessary, please see the reply